Dockerfile

```diff
@@ -0,0 +1,19 @@
+from ruby:2.4.2-jessie
+
+RUN set -ex; \
+	apt-get update; \
+  apt-get install -y --no-install-recommends \
+          qt5-default \
+          libqt5webkit5-dev \
+          gstreamer1.0-plugins-base \
```

Comment:
Why is gstreamer necessary?

Reply:
It's required by [capybara-webkit](https://github.com/thoughtbot/capybara-webkit/wiki/Installing-Qt-and-compiling-capybara-webkit#debian--ubuntu). I will try to figure out a way to build a full featured development/test image and a production image which is based on alpine and lacks those packages.

Reply:
For this pull request, it suffices to focus on either a config for development/test **or** production. It would be ok to base everything on Debian (stretch if possible), if capybara-webkit is a bigger effort on Alpine.

Reply:
Ok, i will evaluate this and create a coherent development/test environment. I think a production environment should be based on the exatly same images so one can rely on the tests(if available ;).

Reply:
Yes, but a production image should not start Puma as primary web server... ;)

Reply:
I switched to stretch and focused on a development/test environment.